shapes/shape.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <QPoint>
+#include <QString>
+#include <string>
+
+namespace jbrush
+{
+class Shape
+{
+public:
+    /* Constructors */
+    Shape(QPoint);
```

Comment:
We need to disable copy instructions here and at the operators below

Reply:
so you're saying to put the deleted members in their respective sections?

```cpp
    /* Constructors */
    Shape(QPoint);
    Shape(const Shape&) = delete;
```